routes/family.js

```diff
@@ -0,0 +1,43 @@
+var express = require('express');
+var router = express.Router();
+var mongoFamily = require('../models/Family').mongoFamily;
+var joiFamily = require('../models/Family').joiFamily;
+
+router.get('/:id', async (req, res) => {
+    
+    try {
+        let id = req.params.id;
+        let family = await mongoFamily.findById(id);
+        res.send(family);
+    } 
+    
+    catch(err) {
+        res.status(400).send({status: "Failed", message: "Cannot get data"})
```

Comment:
you use 'f' for other parts of the code so use it here too
```suggestion
        res.status(400).send({status: "failed", message: "Cannot get data"})
```

Reply:
actually, return null if you can not get data, return the message when there is a database problem